### ISSUE TITLE
Cant install

### ISSUE THREAD

**Issue by theparaodxdoge** [LoukaMB/SynapseX#148]
When i try to install it says it cannot due to my anti virus, it is completely off. And i cant find a contact anywhere so i cant get help.

**Comment by rizkianandafiki** [comment]
same bro

**Comment by IsntLovely** [comment]
Yes synapse X seems to not be working atm keep getting bootstrap error and antivirus completely off also kinda shook that the bladabindi virus is attached to it too.
